Update README.md regarding custom plugins (#1085)
bufbuild/plugins: README.md

Each plugin consists of a `buf.plugin.yaml` file containing metadata and a `Dockerfile` used to build and publish the plugin for remote execution.
Plugins are uniquely identified by the combination of the `name` and `plugin_version` specified in `buf.plugin.yaml` and an auto-generated revision which is incremented each time a modified plugin is pushed to the BSR.

> If you'd like a Protobuf plugin to be added to the Buf Schema Registry, open an issue using the 'Plugin Request for Buf Schema Registry' issue template and our team will follow up. Please note that we are more likely to accept plugins that are stable, well-documented, well-maintained, and have a clear ownership and maintenance story.

Reviewer: ```suggestion
If you'd like a Protobuf plugin to be added to the Buf Schema Registry, open an issue using the 'Plugin Request for Buf Schema Registry' issue template and our team will follow up. Please note that we are more likely to accept plugins that are widely adopted, stable, well-documented, and well-maintained with clear owners.
```
